.github/workflows/tests.yaml

```
@@ -15,10 +15,10 @@ jobs:
       - run: echo "setting versions"
     outputs:
       # JSON encoded array of k8s versions.
-      K8S_VERSIONS: '["1.30.0", "1.29.4", "1.28.9", "1.27.13", "1.26.15"]'
-      VAULT_N: "1.17.3"
-      VAULT_N_1: "1.16.3"
-      VAULT_N_2: "1.15.6"
+      K8S_VERSIONS: '["1.31.2", "1.30.6", "1.29.10", "1.28.9", "1.27.13"]'
```

It looks like there are some slightly newer versions available now for [1.28](https://hub.docker.com/r/kindest/node/tags?name=1.28) and [1.27](https://hub.docker.com/r/kindest/node/tags?name=1.27):

```suggestion
      K8S_VERSIONS: '["1.31.2", "1.30.6", "1.29.10", "1.28.15", "1.27.16"]'
```